woocommerce/woocommerce-ios
# UI: add Fees to Orders

Closes #3084 

This is the fourth and final step to add support for Fees to the Order Details page. (#3084).

* Yosemite: #3466 
* Storage: #3461
* Networking: #3452

This PR adds support for Fees in the UI.

| | Light Appearance | Dark Appearance |
| ---- | ---- | ---- |
| With fees  |  <img src="https://user-images.githubusercontent.com/2722505/104657895-c9bf4680-568f-11eb-8b63-b43bf8e46e99.png" width="350" /> | <img src="https://user-images.githubusercontent.com/2722505/104657900-caf07380-568f-11eb-9674-e055c0b794fb.png" width="350" /> |
| Without fees  |  <img src="https://user-images.githubusercontent.com/2722505/104657902-cb890a00-568f-11eb-8f18-1b302fb70f8e.png" width="350" /> | <img src="https://user-images.githubusercontent.com/2722505/104657903-cb890a00-568f-11eb-9ec7-3057284f650b.png" width="350" /> |

## Changes
* Expose total fees in `OrderPaymentDetailsViewModel`
* Update `LedgerTableViewCell` to add the Fees line
* Move the logic deciding visibility of taxes and coupons to OrderPaymentDetailsViewModel
* Add unit tests to cover the cell and the view model (renaming some of the pre-existing tests to match the naming of other test suites)

## How to test
* Add fees to one or more than one orders in wp-admin. For extra credit, add a single Fee to order A, and more than one Fee to order B
* Checkout the branch, navigate to an order without Fees. The Fees line should not be visible
* Navigate to order A (one Fee) and B (more than one Fee). Notice if the Fees are visible, and it the amount matches the fees assigned in wp-admin. 


Update release notes:
